lib/git.js

```diff
@@ -49,15 +49,15 @@ async function isRefInHistory(ref) {
 }
 
 /**
- * Unshallow the git repository (retriving every commits and tags).
+ * Unshallow the git repository if necessary and fetch all the tags.
  *
  * @param {String} repositoryUrl The remote repository URL.
  */
-async function unshallow(repositoryUrl) {
+async function fetch(repositoryUrl) {
   try {
     await execa('git', ['fetch', '--unshallow', '--tags', repositoryUrl]);
   } catch (err) {
-    debug(err);
+    await execa('git', ['fetch', '--tags', repositoryUrl]);
```

Comment:
if this command fails too, could you log the output?

Reply:
If this command fail the error will be propagated and logged upstream.